scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -330,6 +329,8 @@ get_cluster_configuration() {
         REMOTE_STORAGE_STATUS=1
         REMOTE_LV_SWAP_DEVICE="$(lvs -o path $REMOTE_MD_VOLUMEGROUP | grep $SWAP_DEVICE | awk '{ print $1 }' )"
         REMOTE_LV_MD_DEVICE="$(lvs -o path $REMOTE_MD_VOLUMEGROUP | grep $MD_DEVICE | awk '{ print $1 }')"
+				SINGLE_NODE_RUNNING='--single-node'
```

Comment:
alignment.

Reply:
Had put tabs instead of spaces !!! Making changes.